demo/src/app/components/+carousel/demos/no-pause/no-pause.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-carousel-no-pause',
+  templateUrl: './no-pause.html'
+})
+export class DemoCarouselNoPauseComponent {
+  myInterval = 2500;
+  noWrapSlides = false;
+  showIndicator = true;
```

Comment:
why do you need all these properties? leave only  `noPauseSlides` and rename it to `noPause`